sci-libs/pydicom/pydicom-9999.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_6 python2_7 python3_2 python3_3 python3_4 )
```

Comment:
how do you know? I thought it lists all of them [here](https://github.com/darcymason/pydicom/blob/master/source/setup.py).